CMake/cdat_modules/vtk_external.cmake

```diff
@@ -26,21 +26,21 @@ list(APPEND vtk_build_args
 )
 
 # Use cdat zlib
-if(NOT CDAT_USE_SYSTEM_ZLIB)
-  list(APPEND vtk_build_args
-    -DZLIB_INCLUDE_DIR:PATH=${cdat_EXTERNALS}/include
-    -DZLIB_LIBRARY:FILEPATH=${cdat_EXTERNALS}/lib/libz${_LINK_LIBRARY_SUFFIX}
-  )
-endif()
+#if(NOT CDAT_USE_SYSTEM_ZLIB)
```

Comment:
I have to review bit more but basically if we use system PNG, JPEG etc, then we have to pass the appropriate flags to VTK or else VTK will build its on vtkjpeg, and vtkpng as well. So thing will build but won't run. 

Also, I don't know why we took out cdms?